[2025-02-09T20:27:48.018+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T20:27:48.068+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_to_scrape_and_upload.Upload_data_to_s3 manual__2025-02-09T20:27:43.399181+00:00 [queued]>
[2025-02-09T20:27:48.093+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_to_scrape_and_upload.Upload_data_to_s3 manual__2025-02-09T20:27:43.399181+00:00 [queued]>
[2025-02-09T20:27:48.094+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-09T20:27:48.115+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Upload_data_to_s3> on 2025-02-09 20:27:43.399181+00:00
[2025-02-09T20:27:48.161+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=419) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T20:27:48.165+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_to_scrape_and_upload', 'Upload_data_to_s3', 'manual__2025-02-09T20:27:43.399181+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/scrape_upload_dag.py', '--cfg-path', '/tmp/tmp4ph4cmwu']
[2025-02-09T20:27:48.169+0000] {standard_task_runner.py:72} INFO - Started process 420 to run task
[2025-02-09T20:27:48.170+0000] {standard_task_runner.py:105} INFO - Job 68: Subtask Upload_data_to_s3
[2025-02-09T20:27:48.249+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_to_scrape_and_upload.Upload_data_to_s3 manual__2025-02-09T20:27:43.399181+00:00 [running]> on host 6a5177197bb5
[2025-02-09T20:27:48.368+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_to_scrape_and_upload' AIRFLOW_CTX_TASK_ID='Upload_data_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T20:27:43.399181+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T20:27:43.399181+00:00'
[2025-02-09T20:27:48.371+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T20:28:07.059+0000] {logging_mixin.py:190} INFO - {'error': 'Fileobj must implement read'}
[2025-02-09T20:28:07.068+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-09T20:28:07.080+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T20:28:07.082+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_to_scrape_and_upload, task_id=Upload_data_to_s3, run_id=manual__2025-02-09T20:27:43.399181+00:00, execution_date=20250209T202743, start_date=20250209T202748, end_date=20250209T202807
[2025-02-09T20:28:07.150+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T20:28:07.179+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T20:28:07.182+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
